### ISSUE TITLE
decreased body fat percentage

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/human-phenotype-ontology#5792]
**Preferred term label:**

decreased body fat percentage

**Synonyms**

decreased adipose tissue percentage

**Definition (free text, please give PubMed ID)**

The percentage of fat as a part of total body weight below the norm, usually defined as less than 14% for females and less than 8% for males. (HP:0025521; pennshape.upenn.edu)

**Parent term (use hpo.jax.org/app)**

Growth abnormality (HP:0001507)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

0000-0002-9553-7227
--
